proto/google/events/firebase/testlab/v1/data.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto2";
```

Comment:
APIs should use proto3

Reply:
Sorry I forgot to mention that this PR is only meant for testing. I will revert it now.